src/twisted/test/test_task.py

```diff
@@ -467,6 +467,13 @@ def test_withCountFloatingPointBoundary(self):
         for x in range(count):
             clock.advance(interval)
 
+        # work around https://github.com/python/cpython/issues/100425 on py312
+        def sum(items):
+            total = 0.0
```

Comment:
can we have a different name for this method?

And explain what the function does

```suggestion
        def sum_compat(items):
            """
            Make sure the result is alwasy float.
            On Python 3.11 or older this can be int.
            See: https://github.com/python/cpython/issues/100425
            """
            total = 0.0
```

Reply:
you still get a float back with 3.12 sum, it's just 1.0 instead of 0.9999999999

Reply:
True. I tried to update my suggested comment.